### PR TITLE
Update consul version to 1.10.4

### PR DESCRIPTION
**Pull request by ishustava**
Changes proposed in this PR:
- Update consul version
-

How I've tested this PR:

How I expect reviewers to test this PR:

Checklist:
- [ ] Tests added
- [ ] CHANGELOG entry added 
  > HashiCorp engineers only, community PRs should not add a changelog entry.
  > Entries should use present tense (e.g. Add support for...)

### CODE REVIEW

**Review by thisisnotashwin — APPROVED** [review]
💥

File: charts/consul/values.yaml [hunk 1]
```diff
@@ -85,7 +85,7 @@ global:
   # image: "hashicorp/consul-enterprise:1.10.0-ent"
   # ```
   # @default: hashicorp/consul:<latest version>
-  image: "hashicorp/consul:1.10.3"
+  image: "hashicorp/consul:1.10.4"
```
Comment (ishustava) [inline charts/consul/values.yaml:88]: oops! I think I forgot to update consul version in chart.yaml

Comment (thisisnotashwin) [reply charts/consul/values.yaml:88]: i assumed we would do that when we do the next release release 😅

**Review by t-eckert — APPROVED** [review]

**Pull request merged**